lib/parsers/protected_food_drink_name_parser.rb

```diff
@@ -0,0 +1,229 @@
+class ProtectedFoodDrinkNameParser
```

Comment:
I'd suggest structuring this class and the `ProtectedFoodDrinkNameImportService` one a little differently:

```
lib/importers/protected_food_drink_name.rb -> Importers::ProtectedFoodDrinkName
lib/importers/protected_food_drink_name/parser.rb -> Importers::ProtectedFoodDrinkName::Parser
```

I think because we don't have any other "parsers" or "services" in this app already, having a `lib/parsers` and `lib/services` directory with only a single file in feels a little strange. Plus using the structure above should mean the [autoloader works automatically](https://guides.rubyonrails.org/autoloading_and_reloading_constants.html) and we don't need to do any `require` statements.

Reply:
yes I was unsure on where to put this files :) will move them